src/EditorFeatures/CSharpTest/ForeachToFor/ForeachToForTests.cs

```diff
@@ -0,0 +1,1177 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.ForeachToFor;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.ForeachToFor
+{
+    public partial class ForeachToForTests : AbstractCSharpCodeActionTest
```

Comment:
It's SyntaxKind.ForEachKeyword.  So this should be "ForEachToFor" in all naming.

Reply:
sure.